test/CheckedC/static-checking/bounds-decl-checking-cs.c

```diff
@@ -2,9 +2,8 @@
 // produces warnings about bounds declarations that are not provably
 // true or false.
 //
-// RUN: %clang -cc1 -fcheckedc-extension -verify %s
+// RUN: %clang_cc1 -fcheckedc-extension -verify %s
```

Comment:
Is there a reason to change `%clang -cc1` to `%clang_cc1`? AFAIK both of these are equivalent, no?

Reply:
Apparently, they are not the same. I needed to include <stddef.h>. It does not seem to work with `clang -cc1`. It says it cannot find stddef.h.
Please let me know if there is something better I could do here.

Reply:
`clang` is the driver while `clang_cc1` is the clang frontend. Basically, clang driver will take in all the flags/options and will expand them into a clang -cc1 command line. So if you want to use `clang -cc1` then you would need to provide the path to the includes. So I guess what you did here is correct in that we do not need to go through the `clang` indirection and can directly invoke `clang_cc1`.